history/taskana-simplehistory-provider/src/test/java/pro/taskana/simplehistory/impl/HistoryQueryImplTest.java

```diff
@@ -66,13 +65,12 @@ private HistoryEventImpl createHistoryEvent(
       String workbasketKey,
       String type,
       String userId,
-      String comment,
+      String details,
       Instant created) {
-    HistoryEventImpl he = new HistoryEventImpl(userId);
+    HistoryEventImpl he = new HistoryEventImpl(userId, details);
```

Comment:
what happens to the details?

Reply:
I'm not sure what you mean. HistoryEventImpl extends TaskanaHistoryEvent which has a constructor that sets the details (as well as the userId).